src/main/steps/respondent/review-the-application/post.ts

```diff
@@ -22,7 +23,7 @@ export default class ReviewTheApplicationPostController extends PostController<A
 
     if (req.session.errors.length === 0) {
       try {
-        if (preSubmissionSession.confirmReadPetition === undefined) {
+        if (!preSubmissionSession.confirmReadPetition) {
```

Comment:
This is important RE DB update where the broken cases (will have confirmReadPetition field set to "No" / or Unchecked in FE)